terminal2048/lib/terminal2048/move.ex

```diff
@@ -1,23 +1,74 @@
 defmodule Terminal2048.Move do
-  def move_left(game = %Terminal2048.State) do
+
+  def move_left(game = %Terminal2048.State{}) do
```

Comment:
I'm not diving into thew actual logic here, but on the surface this looks pretty elegant.